src/test/java/com/tradeshift/HelloWorldResourceTest.java

```diff
@@ -0,0 +1,83 @@
+package com.tradeshift;
+
+
+import com.tradeshift.model.DBMessage;
+import com.tradeshift.model.Message;
+import com.tradeshift.resources.HelloWorldResource;
+import com.tradeshift.service.HelloWorldService;
+import com.tradeshift.service.HelloWorldServiceImpl;
+import com.tradeshift.service.MessagesDAO;
+import junit.framework.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+public class HelloWorldResourceTest {
+
+    private MessagesDAO mockMessagesDAO;
+    private HelloWorldService mockHelloWorldService;
+    private HelloWorldResource helloWorldResource;
+
+    @Before
+    public void SetUp() {
+        this.mockMessagesDAO = mock(MessagesDAO.class);
```

Comment:
You can also set up mocks by adding the `@Mock` annotation to the field, and then do `MockitoAnnotations.initMocks(this)` in the setup method.